docs/fractal-example.md

```diff
@@ -0,0 +1,172 @@
+PiCaS fractal example 
+============
+
+
+## Running in Grid
+In this fractal example we will implement the following pilot job workflow:
+
+* First we define and generate the application tokens with all the necessary parameters.
+* Then we define and create a shell script to process one task (*process_task.sh*) that will be sent with the job using the input sandbox. This contains some boiler plate code to e.g. setup the environment, download software or data from the Grid storage, run the application etc. This doesn’t have to be a shell script, however, setting up environment variables is easiest when using a shell script, and this way setup scripts are separated from the application code.
+* We also define and create a Python script to handle all the communication with the token pool server, call the process_task,sh script, catch errors and do the reporting.
+* Finally we define the :abbr:`JDL (Job Description Language)` on the User Interface machine to specify some general properties of our jobs. This is required to submit a batch of pilot jobs to the Grid that will in turn initiate the Python script as defined in the previous step.
+
+
+### Prerequisites
+
+To be able to run the example you must have:
+
+* All the three Grid :ref:`prerequisites` (User Interface machine, Grid certificate, VO membership)
```

Comment:
:ref:`prerequisites` does not work in markdown. Where should this link to? Use `[Link text Here](https://link-url-here.org)"`